### DISCUSSION TITLE
Inclusion Validations

### DISCUSSION THREAD

**Discussion by trevoro** [prisma/prisma#11133] [Q&A] [answered]
Hey Team

I've got a pretty big list of values that I'd like to be able to enforce as valid values in a field in a table.
In pure SQL land I would typically create a table like `countries` with a `code` column and simply reference that column as a way of enforcing a check.

Is there any way to do these inclusion validation tables in Prisma?
Whenever I create a table with a simple `references` column it ends up creating a 1-n mapping which is really not what I want. I'd just like to make the schema enforce valid string options without resorting to using an ENUM or a CHECK constraint.

[This blog post](https://medium.com/swlh/postgresql-3-ways-to-replace-enum-305861e089bc) summarizes a lot of the different approaches pretty well in non-prisma land.

**Comment by samrathchadha** [comment, marked as answer]
There isn't any yet, and I've seen a lot of requests for them, you could open an issue asking for this as a feature.

**Reply by samrathchadha** [reply]
My bad, this: https://www.prisma.io/docs/concepts/components/prisma-client/filtering-and-sorting might work for you
```typescript
const result = await prisma.user.findMany({
  where: {
    email: {
      endsWith: 'prisma.io',
    },
    posts: {
      some: {
        published: true,
      },
    },
  },
  include: {
    posts: {
      where: {
        published: true,
      },
    },
  },
})
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
